docs/support/tutorials/data-mover.md

```diff
@@ -0,0 +1,72 @@
+# Data-mover
+
+Data-mover is a tool to move data between Puhti and Mahti local filesystems and
+Allas and LUMI-O object storage servers, when
+[simple transfers](../faq/how-to-move-data-between-puhti-and-allas.md#move-data-with-rclone)
+are not practical, either because there are many small files, or the size of the
+dataset is large.
+
+We wish the data-mover tool `dm` to be simple to use, and handle all possible
+hard corner cases. It is basically a wrapper around [Restic backup tool](https://restic.readthedocs.io)
+, and stores the data in Restic repositories.
+Restic in turn uses [Rclone](https://rclone.org) for the actual data transfers to
+the object storage servers and back. In addition, the data-mover tool does the
+data transfers in the background, using batch jobs, allowing larger transfers
+than would be practical in regular interactive login sessions.
+
```

Comment:
You should mention, that the data that has been uploaded to Allas this way can't be used with other Allas interfaces like
https://allas.csc.fi, Cyberduck,  rclone or a-commands.

Reply:
Very good point. I think that's the biggest drawback of this approach. You can only access the data with Restic.

Reply:
I changed the description of dm tool so that it explicitly tells that the data is stored in Restic repository _format_.